src/auth/auth.proto

```diff
@@ -392,7 +392,9 @@ message RevokeAuthTokenRequest {
   string token = 1;
 }
 
-message RevokeAuthTokenResponse {}
+message RevokeAuthTokenResponse {
+  int64 num = 1;
```

Comment:
is num a good name here? I know we use Number in other places. Not sure what the right answer is

Reply:
`Number` sounds good!